source/boundary_traction/ascii_data.cc

```diff
@@ -35,14 +35,11 @@ namespace aspect
     void
     AsciiData<dim>::initialize ()
     {
-      const std::map<types::boundary_id,std::shared_ptr<BoundaryTraction::Interface<dim> > >
-      bvs = this->get_boundary_traction();
-      for (typename std::map<types::boundary_id,std::shared_ptr<BoundaryTraction::Interface<dim> > >::const_iterator
-           p = bvs.begin();
-           p != bvs.end(); ++p)
+      std::set<types::boundary_id> traction_bi;
```

Comment:
you introduced a local set here that you are filling but not using.

Reply:
Yes, good point. Stupid copy-paste mistake.